#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_fix_p1.pl_patch_mini_epn.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Call p1.pl from /usr/lib/nagios3 

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' pkg-nagios3~/contrib/mini_epn.c pkg-nagios3/contrib/mini_epn.c
--- pkg-nagios3~/contrib/mini_epn.c	2012-05-07 20:43:10.000000000 +0200
+++ pkg-nagios3/contrib/mini_epn.c	2012-05-07 22:06:49.835073313 +0200
@@ -19,7 +19,7 @@
 	#endif
 	*/
 
-	char *embedding[] = { "", "p1.pl" };
+	char *embedding[] = { "", "/usr/lib/nagios3/p1.pl" };
 	char *plugin_output ;
 	char fname[MAX_INPUT_CHARS];
 	char *args[] = {"", "0", "", "", NULL };
